hugo/layouts/shortcodes/modules_koeln.html

```diff
@@ -1,19 +1,18 @@
+{{ $plotly := resources.Get "js/plotly-1.51.2.min.js" }}
```

Comment:
Mein (möglicherweise naiver) Gedanke, die plotly-Libraries in den Header zu tun war der, dass sowohl die Parisplot (mit den vielen Linien) als auch die noch zu erstellenden Modulplots mit Plotly generiert werden. Lädt der Browser die libraries dann zweimal, wenn beide Plots eingebunden werden?

Reply:
Du hast recht, macht sicherlich mehr Sinn eher d3 und plotly zu bundlen und auf den kommunen seiten einzubinden. So wie es jetzt ist, ist d3 bei den you-draw-it dingern mit drin und plotly ist einzeln.